lib/CL/pocl_file_util.c

```diff
@@ -107,20 +107,6 @@ pocl_exists(const char* path)
   return !access(path, R_OK);
 }
 
-int
```

Comment:
Is it necessary to remove this function even though we don't use it currently?

Reply:
Not really, but it's somewhat dangerous to use, since the reported size isn't correct on some filesystems.